### ISSUE TITLE
Bug: Account switcher still shown when allow_additional_accounts is set to Off

### ISSUE THREAD

**Issue by veitw** [the-djmaze/snappymail#280]
**Describe the bug**
This is a follow-up to #268, as the error message is gone, but the account switcher is still shown in the user drop-down menu when `allow_additional_accounts = Off`. The switch shows the currently logged-in user, just the same as a few pixels above left of the user menu button, and has no function other than wasting screen space.

**To Reproduce**
1. Set `allow_additional_accounts = Off` in `application.ini`.
3. Log in to Snappymail.
4. Click on the user menu.
5. See a useless/functionless menu item with the account of the currently logged-in user.

**Expected behavior**
The list of accounts/account switcher should not be displayed in the user menu when `allow_additional_accounts = Off` is set.

**Screenshots**

This is how it looks:
![grafik](https://user-images.githubusercontent.com/16285895/157332297-4cda1db2-ac58-47a2-bd3c-0929dc51b695.png)

And this is how I think it should look with `allow_additional_accounts = Off` (here enforced using CSS):
![grafik](https://user-images.githubusercontent.com/16285895/157332766-5db0e81d-506f-4ab0-8f4f-1178ef207584.png)

**Desktop (please complete the following information):**
 - Firefox 97
 - SnappyMail 2.13.3

**Comment by the-djmaze** [comment]
It is shown because in mobile (narrow screen) view there is no other way to see which account you logged in.
I might could change it as `hide if width > 1024px`

**Comment by the-djmaze** [comment]
Oops, didn't want to close this.

I have an idea `if not-mobile and not-multiple-accounts`

**Comment by setoy** [comment]
Update: With 2.24.0 it is still kinda visible:
![image](https://user-images.githubusercontent.com/6497266/209311064-8c36c230-22bf-40a3-980d-93298424704c.png)

Most likely due to the css still being applied even though there are no other accounts:
```css
.dropdown-menu *+.dividerbar {
    margin-top: 9px;
    padding-top: 9px;
    border-top: 1px solid var(--border-color,#e5e5e5);
}
```

**Comment by the-djmaze** [comment]
> Removing the `dividerbar` class from the `li` element when no further accounts are present fixes it.

Correct, but i can't just remove the dividerbar. It is used by custom plugins, see #733
